etc/inc/notices.inc

```diff
@@ -350,6 +350,10 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 
 	if ($config['notifications']['smtp']['fromaddress']) {
 		$from = $config['notifications']['smtp']['fromaddress'];
+		preg_match('/[\w\s]+/', $from, $matches);
+		$from_display = trim($matches[0]);
+		if(preg_match('/(?:<)(.+)(?:>)$/', $from, $matches))
+			$from = $matches[1];
```

Comment:
I don't think it's a good idea, it would just add more complication with no gain. You can improve the text that describe the field in that form so users will be aware they can use both formats. I'm going to close this PR. Thanks!